pinot-segment-local/src/main/java/org/apache/pinot/segment/local/data/manager/TableDataManager.java

```diff
@@ -56,6 +56,11 @@ void init(TableDataManagerConfig tableDataManagerConfig, String instanceId,
    */
   void start();
 
+  /**
+   * Stops the table data manager. Should be called only once before shutting down the table data manager.
```

Comment:
Let's add more explanation on the difference between `stop()` and `shutDown()`. Currently the contract around `stop()` is not very clear

Reply:
Makes sense. I was thinking is this enough or should I also add a `_stopped` flag to track the state (and possible reject updates) inside table data manager.